quantum/split_common/split_util.c

```diff
@@ -32,6 +32,16 @@ volatile bool isLeftHand = true;
 
 volatile uint8_t setTries = 0;
 
+// Temporary code to disable JTAG on the slave board
+void disable_JTAG(void) {
```

Comment:
```suggestion
static void disable_JTAG(void) {
```
@vomindoraan you mean like this?

Reply:
Yep, exactly!

Reply:
@That-Canadian This sound good?